tools/RAiDER/delay.py

```diff
@@ -167,7 +167,8 @@ def tropo_delay(dt, wf, hf, args):
                 lons,
                 hgts,
                 pnt_proj,
-                wm_proj
+                wm_proj,
+                flip_xy=pnt_proj.axis_info[0].abbrev in ["X","E","Lon"]
```

Comment:
This is always embedded in geo datasets, so no need to rely on software version?

Reply:
Yeah. This is a function of the datasets / grids. The min version of pyproj that `requirements` is pinned to - this should not be an issue.